COMMUNITY.md

```diff
@@ -15,6 +15,22 @@ Currently on our agenda are:
   - Slack
 - Oversight of [this repository](https://github.com/fluxcd/community).
 
+### Events and Advocacy
+
+It is important to us to create a space for our community to meet, engage, collaborate and learn from each other. Over the past months, we have organised [GitOps Days](https://www.gitopsdays.com/), our participation at CloudNativeCon / KubeCon and at various meetups and WOUG sessions. Check out <https://fluxcd.io/resources/>, to see some of the events we helped to put together recently.
```

Comment:
Can we change "our participation at... to "participated in" and remove "at"
Also, do we need to mention WOUGs without defining it? Saying "meetups" imho includes WOUGs so we can just remove it I think.